postgres/init/22-portal-content.sql

```diff
@@ -115,6 +118,29 @@ COPY portal.app_detail_images (id, app_id, image_url) FROM stdin;
 5cf74ef8-a0a1-0000-a000-000000000007	ac1cf001-7fbc-1f2f-817f-bce05744000b	CE_Marketplace_Image1.png
 5cf74ef8-a0a1-0000-a000-000000000008	ac1cf001-7fbc-1f2f-817f-bce05744000b	CE_Marketplace_Image2.png
 5cf74ef8-a0a1-0000-a000-000000000009	ac1cf001-7fbc-1f2f-817f-bce05744000b	CE_Marketplace_Image3.png
+5cf74ef8-a0a1-0000-a000-0000000000010	5cf74ef8-e0b7-4984-a872-474828beb5d5	Smart Factory-Details1.png
```

Comment:
5cf74ef8-a0a1-0000-a000-0000000000010 is not a valid UUID (it's to long)
(same applies to the ids below)

Reply:
fixed